.circleci/config.yml

```diff
@@ -52,6 +52,19 @@ jobs:
           keys:
             - v1-dependencies-{ { checksum "package.json" }}
 
+      - run: |
+          echo "build --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY}" >> .bazelrc.local
```

Comment:
@sugarmanz Do you know of a way to specify args for any bazel cmd? Instead of having to split it up into `build`, `run`, `test`, etc?

Reply:
I think you can do:

```suggestion
          echo "common --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY}" >> .bazelrc.local
```